web-services/cached-results/src/main/java/datawave/webservice/results/cached/CachedRunningQuery.java

```diff
@@ -842,7 +842,7 @@ private ResultsPage convert(CachedRowSet cachedRowSet, Integer rowBegin, Integer
             while (cachedRowSet.next() && cachedRowSet.getRow() <= rowEnd && !hitPageByteTrigger) {
                 if (log.isTraceEnabled())
                     log.trace("CRS.position: " + cachedRowSet.getRow() + ", size: " + cachedRowSet.size());
-                CacheableQueryRow row = CacheableQueryRowReader.createRow(cachedRowSet, this.fixedFieldsInEvent);
+                CacheableQueryRow row = CacheableQueryRowReader.createRow(cachedRowSet, this.fixedFieldsInEvent, this.queryLogic.getResponseObjectFactory());
```

Comment:
Instead of `this.queryLogic.getResponseObjectFactory()` could/should we do `this.responseObjectFactory` instead? (This does bring up a good point, are there places where we should be getting the responseObjectFactory from the query logic instead of injecting it directly?)

Reply:
ah, right, sure